pkg/arm/v13/data/master-startup.sh

```diff
@@ -65,7 +65,8 @@ if ! grep /var/lib/etcd /etc/fstab; then
   restorecon -R /var/lib/etcd
 fi
 
-for dst in tcp,8444; do
+# accepting 8443 for the admission controller
+for dst in tcp,8443 tcp,8444; do
```

Comment:
:8443 is accessible from within the cluster but is protected by TLS client certificate.  It is not in the NSG so is not accessible from outside the cluster.